packages/fiori-freestyle-writer/src/index.ts

```diff
@@ -13,16 +13,34 @@ import { initI18n } from './i18n';
 import { getBootstrapResourceUrls, getPackageScripts } from '@sap-ux/fiori-generator-shared';
 import { getTemplateVersionPath, processDestinationPath } from './utils';
 import { applyCAPUpdates, type CapProjectSettings } from '@sap-ux/cap-config-writer';
+import type { Logger } from '@sap-ux/logger';
+import { generateFreestyleOPAFiles } from '@sap-ux/ui5-test-writer';
 
+/**
+ * Adds test scripts to the package.json object.
+ *
+ * @param {Package} packageJson - The package.json object to update.
+ * @param {boolean} addMock - Whether to include the UI5 mock YAML configuration.
+ */
+function addTestScripts(packageJson: Package, addMock: boolean): void {
```

Comment:
Please consider moving this to another function in another file along with all the test related code.